packages/buffetjs-styles/src/components/List/index.js

```diff
@@ -12,7 +12,7 @@ const List = styled.div`
   width: 100%;
   position: relative;
   overflow-x: scroll;
-  border-radius: 3px;
+  border-radius: 2px;
```

Comment:
It might be a good idea to add the `borderRadius` property to `@buffetjs/styles` `assets/size` so we can use it in the other packages.